tvb_documentation/manuals/UserGuide/DataExchange.rst

```diff
@@ -190,24 +194,6 @@ In correlation with export operations, |TVB| interface allows import of data in
 |TVB| data type. Depending on the uploaded file format, imported data can be as
 follows:
 
-
-File Format
-~~~~~~~~~~~
-
-1. If user uploads a ZIP file, the system automatically assumes a datatype group
-   must be imported and then process the file accordingly. More specifically, it
-   tries to find an XML file, within the ZIP file, describing the operation(s)
-   that generated the data types and the list of HDF5 files for each datatype.
-
-#. If user uploads a simple HDF5/H5 file, the system assumes that a simple data
-   type is imported and tries to process the file accordingly. Basically it
-   reads the metadata stored in the root node group and determines the data type
-   (e.g. connectivity, time series ...). Based on the detected type of data, the
-   rest of the details are filled and the object is stored in the database.
-
```

Comment:
Why take this info out? We should add the case of importing the ZIP from Export Datatype with links